### PR TITLE
New sort control + result counter

### PR DESCRIPTION
**Pull request by isoos**
Instead of the style down-arrow, I'm using a simple one, tracking it in #3246.

Screen Shot 2020-01-22 at 16 25 16

### CODE REVIEW

File: app/lib/frontend/templates/listing.dart [hunk 1]
```diff
@@ -122,12 +122,13 @@ String renderPkgIndexPage(
   final values = {
     'sort_control_html': renderSortControl(searchQuery),
     'is_search': isSearch,
-    'title': title ?? topPackages,
     'package_list_html': renderPackageList(packages, searchQuery: searchQuery),
     'has_packages': packages.isNotEmpty,
     'pagination': renderPagination(links),
-    'search_query': searchQuery?.query,
     'total_count': totalCount,
+    // TODO: remove the keys below after we have migrated to the new design
```
Comment (sigurdm) [inline app/lib/frontend/templates/listing.dart]: Would it make sense to mark these with the issue number to make these TODO's easier to find?
```
TODO(3246): remove the keys below after we have migrated to the new design
```

File: pkg/web_css/lib/src/_list_experimental.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/* non-indented rule to restrict the content of this block to the experimental pages */
```
Comment (sigurdm) [inline pkg/web_css/lib/src/_list_experimental.scss]: I think this needs a copyright header

File: pkg/web_css/lib/src/_list.scss [hunk 3]
```diff
@@ -1,3 +1,26 @@
+/* non-indented rule to restrict the content of this block to the non-experimental pages */
```
Comment (sigurdm) [inline pkg/web_css/lib/src/_list.scss]: I think this needs a copyright header

File: app/lib/frontend/templates/listing.dart (hunk 1 above)
Comment (isoos) [reply app/lib/frontend/templates/listing.dart]: Done. I'll update the earlier similar TODOs if I touch the code nearby.

File: pkg/web_css/lib/src/_list_experimental.scss (hunk 2 above)
Comment (isoos) [reply pkg/web_css/lib/src/_list_experimental.scss]: Done. I'll update the other SCSS files as I touch the styles in them.

**Pull request merged**